AppKit/CPTextView/CPTextView.j

```diff
@@ -0,0 +1,1890 @@
+    int                         _stickyXLocation;
+
+    CPArray                     _selectionSpans;
+}
+
+
+#pragma mark -
+#pragma mark Class methods
+
+/* <!> FIXME
+    just a testing characterSet
+    all of this depend of the current language.
+    Need some CPLocale support and maybe even a FSM...
+ */
+
+#pragma mark -
+#pragma mark Init methods
+
+- (id)initWithFrame:(CGRect)aFrame textContainer:(CPTextContainer)aContainer
+{
+    if (self = [super initWithFrame:aFrame])
+    {
+        [self _init];
+        [aContainer setTextView:self];
+    }
+
+    [self registerForDraggedTypes:[CPColorDragType]];
+
+    return self;
+}
+
+- (id)initWithFrame:(CGRect)aFrame
+{
+    var container = [[CPTextContainer alloc] initWithContainerSize:CGSizeMake(aFrame.size.width, 1e7)];
+
+    return [self initWithFrame:aFrame textContainer:container];
+}
+
+- (void)_init
+{
+#if PLATFORM(DOM)
+        _DOMElement.style.cursor = "text";
+#endif
+
+    _selectionRange = CPMakeRange(0, 0);
+    _textContainerInset = CGSizeMake(2, 0);
+    _textContainerOrigin = CGPointMake(_bounds.origin.x, _bounds.origin.y);
+
+    [self setEditable:YES];
+    [self setSelectable:YES];
+
+    _isFirstResponder = NO;
+    _delegate = nil;
+    _delegateRespondsToSelectorMask = 0;
+
+    _selectionGranularity = CPSelectByCharacter;
+    _selectedTextAttributes = [CPDictionary dictionaryWithObject:[CPColor selectedTextBackgroundColor]
+                                            forKey:CPBackgroundColorAttributeName];
+
+    _insertionPointColor = [CPColor blackColor];
+    _textColor = [CPColor blackColor];
+    _font = [CPFont systemFontOfSize:12.0];
+    [self setFont:_font];
+    [self setBackgroundColor:[CPColor whiteColor]];
+
+    _typingAttributes = [[CPDictionary alloc] initWithObjects:[_font, _textColor] forKeys:[CPFontAttributeName, CPForegroundColorAttributeName]];
+
+    _minSize = CGSizeCreateCopy(_frame.size);
+    _maxSize = CGSizeMake(_frame.size.width, 1e7);
+
+    [self setRichText:NO];
+    _usesFontPanel = YES;
+    _allowsUndo = YES;
+    _isVerticallyResizable = YES;
+    _isHorizontallyResizable = NO;
+
+    _caretRect = CGRectMake(0, 0, 1, 11);
+}
+
+
+#pragma mark -
+#pragma mark Copy and past methods
+
+- (void)copy:(id)sender
+{
+   _copySelectionGranularity = _previousSelectionGranularity;
+   [super copy:sender];
+}
+
+- (void)paste:(id)sender
+{
+    if (_copySelectionGranularity > 0)
+    {
+        if (![self _isCharacterAtIndex:MAX(0, _selectionRange.location - 1) granularity:_copySelectionGranularity])
+            [self insertText:" "];
+    }
+    [super paste:sender];
+
+    if (_copySelectionGranularity > 0)
+    {
+        if (![self _isCharacterAtIndex:CPMaxRange(_selectionRange) granularity:_copySelectionGranularity])
+            [self insertText:" "];
+    }
+}
+
+#pragma mark -
+#pragma mark Responders method
+
+- (BOOL)acceptsFirstResponder
+{
+    if ([self isSelectable])
+        return YES;
+
+    return NO;
```

Comment:
Just return [self isSelectable] right now.

I'm surprised that this method does not have the same behavior as the one of the CPTextField. You you check